core-bundle/src/Resources/contao/modules/ModuleLogin.php

```diff
@@ -72,7 +72,7 @@ public function generate()
 		{
 			$this->targetPath = base64_decode($request->request->get('_target_path'));
 		}
-		elseif ($this->redirectBack && $request && $request->query->has('redirect'))
+		elseif ($this->redirectBack && $request->query->has('redirect'))
```

Comment:
This is a nice one. We do check if `$request` is not `null` in the else condition but not in the if condition. Therefore I have removed the check completely. Would you prefer to add it again?

Reply:
Fine for me, the error when `$request` is `null` would occur on line 71 anyway.

Reply:
We would have to add the check there as well if we decide to add it again.

Reply:
Added in bcbd2ad02e32f42a4c41734073ccc59f9c125bf7 again.